README.md

```diff
@@ -4,6 +4,19 @@ Tools to help you pick a prior.
 
 **This package is very new and all of its features are experimental, not yet well tested, and subject to change without notice.**
 
+## The Zen of PreliZ
+* Being open source. 
```

Comment:
Do we want to be open in other ways? like being also community-driven, being diverse?